Steganography-App-CMSC-495/steganography_backend
# Steg 4 - Added image saving support on custom image data

**Added**
- Core function to save image from structure obtained (and modified) from getImageData

**Changed**
- Driver changed to separate tests into methods
- Image save test doubles the size of the pixel_test.png

**QA**
Run the core driver as usual (python src/util_driver.py)
View pixel output data between the image load test and image save test to verify image was resized properly, followed by viewing the image itself in any editor, original is 2x2 pixels, new version should be 4x4.
